nixos/modules/services/desktops/gnome/gnome-keyring.nix

```diff
@@ -24,14 +26,23 @@
         '';
       };
 
+      enableSSHSupport = lib.mkOption {
```

Comment:
This can be a `mkEnableOption`. 
```nix
enableSSHSupport = lib.mkEnableOption "SSH agent support for GNOME Keyring by setting the SSH_AUTH_SOCK environment variable";
```